src/containers/TopNav/TopNavMobileMenu/index.tsx

```diff
@@ -41,14 +41,14 @@ const TopNavMobileMenu = () => {
           {title}
         </button>
         {smallDevice && (
-          <div className="TopNavMobileMenu__icon-holder">
```

Comment:
Instead of making both the title and icon clickable individually, how about just make the outer div clickable?

Reply:
Okay then. I'll do that instead